ccmlib/cmds/cluster_cmds.py

```diff
@@ -541,6 +544,8 @@ def get_parser(self):
 
     def validate(self, parser, options, args):
         Cmd.validate(self, parser, options, args, load_cluster=True)
+        if self.options.no_wait and self.wait_for_binary_proto:
+            print_("WARN: Both --wait-for-binary-proto and --no-wait were specified. --no-wait will be preferred.")
```

Comment:
Sorry, I mean this:

https://github.com/mambocab/ccm/commit/0aa0891fc3d163b220ec36514cb2ee3b39dc3de2

or... something like that. I, um, haven't actually run it, and I don't remember how argparse works right now.